transport-native-epoll/src/main/java/io/netty/channel/epoll/LinuxSocket.java

```diff
@@ -154,7 +154,7 @@ public static LinuxSocket newSocketDomain() {
     private static native int getTcpUserTimeout(int fd) throws IOException;
     private static native int isIpFreeBind(int fd) throws IOException;
     private static native int isIpTransparent(int fd) throws IOException;
-    private static native void getTcpInfo(int fd, int[] array) throws IOException;
+    private static native void getTcpInfo(int fd, long[] array) throws IOException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)